### PR TITLE
ft: Messages crud operations

### PR DESCRIPTION
**Pull request by Edmondus**
#### What does this PR do?
[what does this PR do?]
>Read and save messages
#### Description of Task to be completed
[the tasks that you completed on the feature]
>view all messages
>save a message

>localhost:3000/messages to read all messages
>localhost:3000/messages/saveMessage to save one message

### CODE REVIEW

File: server/controllers/articles.js [hunk 1]
```diff
@@ -1 +1,129 @@
...
+            success: false,
+            status: 500,
+            error: 'Server error'
+        });
+    }
+}
+
+export const createArticle = async (req, res, next) => {
+        const newArticle = new Article (
+            {
```
Comment (habinezadalvan) [inline server/controllers/articles.js]: How do you validate your inputs? If you forgot to validate, you can use the **JOI** package to validate.

File: server/controllers/articles.js [hunk 2]
```diff
@@ -1 +1,129 @@
...
+                });
+            }
+        });
+        
+}
+
+export const getOneArticle = async (req, res, next) => {
+    try {
+        const article = await Article.findById(req.params._id);
+console.log('>>>>>',req.params._id);
```
Comment (habinezadalvan) [inline server/controllers/articles.js]: Could you please remove this unnecessary console.log?

**Review by habinezadalvan — COMMENTED** [review]
It looks good, however I think for better git workflow; it would be better if you separate the functionalities into separate branches. Again, check for the comments below and work on them before merging the work.

File: server/controllers/articles.js (hunk 1 above)
Comment (Edmondus) [reply server/controllers/articles.js]: I was planning to create middlewares and validate my inputs using regular expressions.
but I'll try that JOI package.

File: server/controllers/articles.js (hunk 2 above)
Comment (Edmondus) [reply server/controllers/articles.js]: Yeah sure!

**Pull request closed without merging**